Common/Systems/Experience/ExperienceSystem.cs

```diff
@@ -0,0 +1,168 @@
+﻿using System.Collections.Generic;
+using PathOfTerraria.Common.Systems.Networking.Handlers;
+using Terraria.ID;
+
+namespace PathOfTerraria.Common.Systems.Experience;
+
+// ReSharper disable once ClassNeverInstantiated.Global
+public class ExperienceTracker : ModSystem
+{
+	private static Mechanics.Experience[] _trackedExp;
```

Comment:
Shouldn't Mechanics just be add to using, having "Mechanics.Experience", like, 30 times seems kinda weird.

Reply:
@shnakamura

Reply:
I'm not really sure why this has come to my attention since I havent made this system, but sure.

Reply:
I don't believe I touched these files so this would have been done by you.

Reply:
Seems like its just always been like that, check blame if there's any remainining doubts.

Reply:
Yeah, likely a mistake on my end, let's just say that it's fine, going to merge as is, then - but I do still wonder about the player ting above.